sample-compose/app/src/main/java/co/nimblehq/sample/compose/ui/screens/home/HomeScreen.kt

```diff
@@ -44,12 +47,28 @@ fun HomeScreen(
     )
 }
 
+@OptIn(ExperimentalPermissionsApi::class)
 @Composable
 private fun HomeScreenContent(
     uiModels: List<UiModel>,
     showLoading: IsLoading,
     onItemClick: (UiModel) -> Unit
 ) {
+    val cameraPermissionState = rememberPermissionState(CAMERA)
+    if (cameraPermissionState.status.isGranted) {
+        Timber.d("${cameraPermissionState.permission} granted")
+    } else {
+        if (cameraPermissionState.status.shouldShowRationale) {
+            Timber.d("${cameraPermissionState.permission} needs rationale")
+        } else {
+            Timber.d("Request cancelled, missing permissions in manifest or denied permanently")
+        }
+
+        LaunchedEffect(Unit) {
```

Comment:
@luongvo May I know why we need to call `LaunchedEffect` inside `else` block here 🤔

Reply:
@Wadeewee we need to request to show the `permission request dialog` for the user to approve the required permission in case of `shouldShowRationale` or `denied` 🤓